lm_eval/tasks/french_bench/french_bench_wikitext_fr.yaml

```diff
@@ -0,0 +1,24 @@
+group:
+  - french_bench_perplexity
+task: french_bench_wikitext_fr
+dataset_path: asi/wikitext_fr
+dataset_name: wikitext-35
+output_type: loglikelihood_rolling
+training_split: train
+validation_split: validation
+test_split: test
+doc_to_text: ""
```

Comment:
```suggestion
num_fewshot: 0
doc_to_text: ""
```

Reply:
awesome, wasn't aware of the arg, thanks !